### DISCUSSION TITLE
Problem at Installation

### DISCUSSION THREAD

**Discussion by STRIDER1512** [Security-Onion-Solutions/securityonion#13909] [Unsupported Versions]
While I am installing in the standalone mode I am getting this issue that could not resolve the host is it because of the ip management or something else any suggestions will be helpful,after this the installation process exits ,![soc-2024-11-06-14-52-06](https://github.com/user-attachments/assets/97fdfca2-784e-4cac-9e5c-8bb2adee4f0b)

**Comment by rh-ops** [comment]
Id recommend checking your DNS settings and making sure your firewall isn't blocking any connections SO is trying to make.

**Reply by STRIDER1512** [reply]
Thanks for the reply I think the problem is from the dns side,firewall its clear.

**Reply by NeoLife-iT** [reply]
It took me awhile to get the installation done - over 40 times :-) .  My new corp network yeild similar error until I changed the DNS from default 8.8.8.8, 8.8.4.4  to local network..try your ISP see if that work.  BTW the defaut dns work fine in my home lab.  I am using hyper-v

**Reply by rh-ops** [reply]
> Thanks for the reply I think the problem is from the dns side,firewall its clear.

Heres a few more things you can also try to diagnose the issue

Check that it ping itself (local host 127.0.0.1)
Check if it can ping the gateway
Check to make sure the management NIC and monitoring NIC are correct and not swapped

**Reply by STRIDER1512** [reply]
> It took me awhile to get the installation done - over 40 times :-) .  My new corp network yeild similar error until I changed the DNS from default 8.8.8.8, 8.8.4.4  to local network..try your ISP see if that work.  BTW the defaut dns work fine in my home lab.  I am using hyper-v
> 
> 
> 
> 

Completed the installation more than 10 times with same error deleting the instance in vmware,I am assigning the default dns only so I will change it and let you know,thanks
